src/main/g8/$moduleName$/test/src/example/HelloSuite.scala

```diff
@@ -3,11 +3,11 @@ package example
 class HelloSuite extends munit.FunSuite {
   test("numbers") {
     val obtained = 42
-    val expected = 43
```

Comment:
This was kind of supposed to show a broken test :sweat_smile:

Reply:
Ah! I can take out the check for `__.test` if you'd rather keep it this way? ha, I just wanted to show that the validation targets work. I was also thinking of adding a way to allow for a target to fail when expected, but I figured most templates wanted everything working out of the box.

Reply:
It's totally fine, it's really not important, people can easily figure out.